README.md

```diff
@@ -36,17 +28,31 @@ Then put the given 'server address' (including that last `/`) into the address b
 
 ## Configuration
 
-TODO:document configuation options
+Site variables can be overriden in the `_config.yml` on the response site. We will have a separate response site template repository we will document the configuation setting overrides.
 
 ## Known issues
 
 See our [open issues](https://github.com/hurricane-response/michael_website/issues)
 
 **Note:** Mapbox donates their product to us for free for volunteer projects only -- if the project ends up getting adopted by a government partner or somehow turns into a commercial project, we'll have to plan a transition to a different Mapbox account.
 
+## FAQ
+
+### Why Jekyll and GitHub?
+
+We wanted a site up fast, the volunteer team that started working on Hurricane Florence knew Jekyll, and we wouldn't have to worry about hosting and infrastructure. As we moved away from an iframed React app running code and on a domain we didn't control, we moved to fully using Jekyll and Mapbox.
+
+### Why a Jekyll Theme?
+
+The problem we are trying to solve with a gem based theme is prevent duplicate work and expidite the process to lauch a new response site. After Florence we realized we were having to copy files over to a new repository so we could customize the name and map center among other reasons. Any new work added to a new site wouldn't get added to an existing response site. Not to mention, there wasn't any continuity in tracking issues or properly crediting contributions.
+
+Jekyll introduced [gem based themes](https://jekyllrb.com/docs/themes/) in 3.5. Subsequently around 3.6.2 release Ben Balter [released a plugin](https://github.com/benbalter/jekyll-remote-theme) that allows GitHub Pages hosted sites to use any gem based theme that was public on GitHub. The idea is, if we keep the presentational markup and base map code in a theme repository that each response site can use by overriding variables in the `_config.yml` file, it would be quick and easy to deploy a new response site while keeping all the code/issues/contributions in a single repository.
+
 ## Getting involved
 
-Our [getting started](https://bit.ly/2N6YVYD) document serves as the best entry for the project for all types of contributions.
+Our [getting started](https://docs.google.com/document/d/1f_ODFvGzrOihfkckSxJ8hU4d9I3b389Rx4aAEFEHvTw/edit?usp=sharing) document serves as the best entry for the project for all types of contributions.
+
+Also see our [Contributing file](Contributing.md) for repo specific contributions.
 
 ## Open source licensing info
 1. [LICENSE](LICENSE)
```

Comment:
There's no `LICENSE` file in the repo that I can find. We should add one. Once that's there, I'd change this to read:

```
This software is licensed under the ... license. For more details, see the [LICENSE](LICENSE) file.
```

I'll suggest MIT license, but I'm open to discussion on that point.

Reply:
I would second the use of MIT.  If you look under Insight > Community, you can add a LICENSE text very easily.

Reply:
I'm fine with the most open license available for the theme code. I believe we used MIT for the past themes.